### PR TITLE
get kafka_brokers_sasl from instance.extensions

### PR DESCRIPTION
**Pull request by JunliWang**
### Description

request from customer
```
 
It is a terraform feature request.
 
So we need to get bootstrap cluster when cluster deployment is terminated.
 
To get bootstrap cluster (variable kafka_brokers_sasl) we must create a first topic. If we don’t create a topic on cluster.. kafka_brokers_sasl variable is empty.
 
So we would like to be able to get variable kafka_brokers_sasl without having to create a first topic.
```

### Release required?

- [ ] No release
- [x] Patch release (`x.x.X`)
- [ ] Minor release (`x.X.x`)
- [ ] Major release (`X.x.x`)

### Run the pipeline

If the CI pipeline doesn't run when you create the PR, the PR requires a user with GitHub collaborators access to run the pipeline.

Run the CI pipeline when the PR is ready for review and you expect tests to pass. Add a comment to the PR with the following text:

```
/run pipeline
```

### Checklist for reviewers

- [ ] If relevant, a test for the change is included or updated with this PR.
- [ ] If relevant, documentation for the change is included or updated with this PR.

### For mergers

- Use a conventional commit message to set the release level. Follow the [guidelines](https://terraform-ibm-modules.github.io/documentation/#/merging.md).
- Include information that users need to know about the PR in the commit message. The commit message becomes part of the GitHub release notes.
- Use the **Squash and merge** option.

### CODE REVIEW

**Comment by Ak-sky** [comment]
PR pipeline failing with below error, waiting on this [fix](https://github.com/terraform-ibm-modules/terraform-ibm-event-streams/pull/312) to get merged.

```
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │ summary: "resourceIBMEventStreamsTopicCreate createSaramaAdminClient: kafka: client
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   has run out of available brokers to talk to: 6 errors occurred:\n\t* kafka server:
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   SASL Authentication failed: Authentication failed, invalid credentials\n\t* kafka
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   server: SASL Authentication failed: Authentication failed, invalid credentials\n\t*
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   kafka server: SASL Authentication failed: Authentication failed, invalid credentials\n\t*
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   kafka server: SASL Authentication failed: Authentication failed, invalid credentials\n\t*
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   kafka server: SASL Authentication failed: Authentication failed, invalid credentials\n\t*
TestRunUpgradeExample 2024-10-25T06:12:53Z logger.go:66: │   kafka server: SASL Authentication failed: Authentication failed, invalid credentials\n"
```

File: .gitignore [hunk 1]
```diff
@@ -54,3 +54,5 @@ Brewfile.lock.json
 
 # Visual Studio Code
 .vscode/
+
+tests/vendor
```
Comment (Ak-sky) [inline .gitignore:58]: Why are we adding this?

Comment (JunliWang) [reply .gitignore:58]: I was trying to run `go mod tidy`, `go mod vendor` and then `go test` .
tests/vendor should not be committed.

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**